src/ServicePulse.Host/vue/src/composables/serviceNotifications.ts

```diff
@@ -33,8 +35,16 @@ export async function useUpdateEmailNotifications(settings: UpdateEmailNotificat
 export async function useTestEmailNotifications() {
   try {
     const response = await usePostToServiceControl("notifications/email/test");
+
+    let responseStatusText;
+    if (useIsSupported(environment.sc_version, "5.2.0")) {
```

Comment:
Does this take into account 5.2.0 and higher?

Reply:
Good catch

```suggestion
    if (useIsSupported(environment.sc_version, "5.2")) {
```

Reply:
I checked the logic and I think it would have worked before too since it checks every version you pass against the current version. I don' t know if it can deal with X.Y version only honestly.

Reply:
I would say revert this for now since we are always checking against X.Y.Z in other code paths.